diff --git a/common/xbps-src/shutils/common.sh b/common/xbps-src/shutils/common.sh
index 663478ea..9e8f1566 100644
--- a/common/xbps-src/shutils/common.sh
+++ b/common/xbps-src/shutils/common.sh
@@ -624,7 +624,8 @@ setup_pkg() {
         unset CC_host CXX_host CPP_host GCC_host FC_host LD_host AR_host AS_host
         unset RANLIB_host STRIP_host OBJDUMP_host OBJCOPY_host NM_host READELF_host
         unset CFLAGS_host CXXFLAGS_host CPPFLAGS_host LDFLAGS_host FFLAGS_host
-        unset RUSTFLAGS
+        # Rust flags which are passed to rustc
+        export RUSTFLAGS="$XBPS_RUSTFLAGS"
     fi
 
     # Setup some specific package vars.
