--- Makefile.in	2011-10-21 19:15:54.000000000 +0100
+++ Makefile.in	2021-07-13 05:03:57.585610221 +0100
@@ -681,33 +681,6 @@
 	$(CC) $(CFLAGS) $(srcdir)/mycb.c -o mycb
 
 all: startup.yss
-	@ENABLE_GECODE@ @INSTALL_DLLS@ (cd library/gecode; $(MAKE))
-	@INSTALL_DLLS@ (cd library/lammpi; $(MAKE))
-	@INSTALL_MATLAB@ (cd library/matlab; $(MAKE))
-	@INSTALL_DLLS@ (cd library/matrix; $(MAKE))
-	@INSTALL_DLLS@ (cd library/random; $(MAKE))
-	@INSTALL_DLLS@ (cd library/regex; $(MAKE))
-	@INSTALL_DLLS@ (cd library/rltree; $(MAKE))
-	@ENABLE_WINCONSOLE@ (cd LGPL/swi_console; $(MAKE))
-	@INSTALL_DLLS@ (cd library/system; $(MAKE))
-	@INSTALL_DLLS@ (cd library/tries; $(MAKE))
-	@ENABLE_CLIB@ @INSTALL_DLLS@ (cd packages/clib; $(MAKE))
-	@ENABLE_HTTP@ @INSTALL_DLLS@ (cd packages/http; $(MAKE))
-	@ENABLE_PLDOC@  @INSTALL_DLLS@ (cd packages/pldoc; $(MAKE))
-	@ENABLE_PLUNIT@ @INSTALL_DLLS@ (cd packages/plunit; $(MAKE))
-	@ENABLE_R@ @INSTALL_DLLS@ (cd packages/R; $(MAKE))
-	@ENABLE_RDF@ @INSTALL_DLLS@ (cd packages/RDF; $(MAKE))
-	@ENABLE_SEMWEB@ @INSTALL_DLLS@ (cd packages/semweb; $(MAKE))
-	@ENABLE_SGML@ @INSTALL_DLLS@ (cd packages/sgml; $(MAKE))
-	@ENABLE_YAPR@ (cd packages/YapR; $(MAKE))
-	@ENABLE_CLPBN_BP@ (cd packages/CLPBN/clpbn/bp ; $(MAKE))
-	@ENABLE_MINISAT@ (cd packages/swi-minisat2/C; $(MAKE))
-	@ENABLE_ZLIB@ @INSTALL_DLLS@ (cd packages/zlib; $(MAKE))
-	@ENABLE_CPLINT@ (cd packages/cplint/approx/simplecuddLPADs; $(MAKE))
-	@ENABLE_CPLINT@ (cd packages/cplint; $(MAKE))
-	@ENABLE_CUDD@ (cd packages/ProbLog/simplecudd; $(MAKE))
-	@ENABLE_CUDD@ (cd packages/ProbLog/simplecudd_lfi; $(MAKE))
-	@ENABLE_JPL@ @INSTALL_DLLS@ (cd packages/jpl; $(MAKE))
 
 startup.yss: yap@EXEC_SUFFIX@ $(PL_SOURCES)
 	-rm -f startup.yss
