--- C/alloc.c	2011-10-21 19:15:54.000000000 +0100
+++ C/alloc.c	2021-07-13 04:34:44.657658975 +0100
@@ -946,7 +946,7 @@
 	   MAP_PRIVATE | MAP_ANON, -1, 0);
 #endif
 #else
-  fd = open("/dev/zero", O_RDWR, S_IRUSR | S_IWUSR | S_IRGRP | S_IROTH);
+  fd = -1;
   if (fd < 0) {
 #if HAVE_MKSTEMP
     char file[256];
@@ -968,7 +968,7 @@
     itos(getpid(), &file[12]);
 #endif /* HAVE_TMPNAM */
 #endif /* HAVE_MKSTEMP */
-    fd = open(file, O_CREAT|O_RDWR, S_IRUSR | S_IWUSR | S_IRGRP | S_IROTH);
+    fd = open(file, O_CREAT|O_RDWR, S_IXUSR | S_IRUSR | S_IWUSR | S_IRGRP | S_IROTH);
     if (fd < 0) {
       Yap_Error(FATAL_ERROR, TermNil, "mmap could not open %s", file);
       return NULL;
@@ -1042,7 +1042,7 @@
    a = mmap(base, (size_t) s, PROT_READ | PROT_WRITE | PROT_EXEC,
 	    MAP_PRIVATE | MAP_ANON | fixed_allocation, -1, 0);
 #else
-   fd = open("/dev/zero", O_RDWR);
+   fd = -1;
    if (fd < 0) {
 #if HAVE_MKSTEMP
      char file[256];
@@ -1068,7 +1068,7 @@
      itos(getpid(), &file[12]);
 #endif /* HAVE_TMPNAM */
 #endif /* HAVE_MKSTEMP */
-    fd = open(file, O_CREAT|O_RDWR, S_IRUSR | S_IWUSR | S_IRGRP | S_IROTH);
+    fd = open(file, O_CREAT|O_RDWR, S_IXUSR | S_IRUSR | S_IWUSR | S_IRGRP | S_IROTH);
     if (fd < 0) {
       LOCAL_ErrorMessage = LOCAL_ErrorSay;
       snprintf4(LOCAL_ErrorMessage, MAX_ERROR_MSG_SIZE,
